### params for IBVS nodes ###
### Waypoints ###

ibvs: {

p_des: [-200, -200, 200, -200, 200, 200, -200, 200]

}

ibvs_inner: {

p_des: [-150, -150, 150, -150, 150, 150, -150, 150]

}
